civictechro/fiipregatit.ro
# feature(backend) Finalized work on guides

#### Rezumat al schimbărilor:
- Update Wordpress from 4.9 to 4.9.2
- Update ACF from 4.4.12 to 5.6.7

- Rebuilt the guide structure to match reqs
- Overwrote import/export files with new versions
- Worked on the custom entity code to match the new structure
- Slight refactoring on the custom lib code
- Small fixes on the frontend

#### Test plan:

- Load admin, make sure it works
- Load homepage, make sure it works
- Test minimal req. custom fields
